src/AccessibleGooglePlacesAutocomplete.tsx

```diff
@@ -101,9 +100,31 @@ export class AccessibleGooglePlacesAutocomplete extends React.Component<
     this.getStatusNoResultsMessage = this.getStatusNoResultsMessage.bind(this);
   }
 
+  public onAutoCompleteSelect = (value: string) => {
+    this.predictions.forEach(element => {
+      if (element.description === value) {
+        const placesService = new google.maps.places.PlacesService(
```

Comment:
This creates a new service instance each time an address is selected. This could cause performance issues. Could the places service be created at the same time as the autocomplete service?